File: code/python/hello.py
import mysql.connector

def main():
    print("Hello, World! From Python.")
    try:
        connection = mysql.connector.connect(
            host="mysql",
            port=3306,
            database="testdb",
            user="testuser",
            password="testpass"
        )
        cursor = connection.cursor()
        cursor.execute("SELECT 'Hello from MySQL' AS message")
        for (message,) in cursor:
            print(message)
    except Exception as e:
        print("Error:", e)
    finally:
        if connection.is_connected():
            connection.close()

if __name__ == "__main__":
    main()

File: code/js/hello.js
const mysql = require('mysql2');

console.log("Hello, World! From JavaScript.");

const connection = mysql.createConnection({
    host: "mysql",
    port: 3306,
    user: "testuser",
    password: "testpass",
    database: "testdb"
});

connection.connect(err => {
    if (err) {
        console.error("Error connecting to MySQL:", err);
        return;
    }
    connection.query("SELECT 'Hello from MySQL' AS message", (error, results) => {
        if (error) throw error;
        results.forEach(row => console.log(row.message));
        connection.end();
    });
});
